_posts/2017-01-30-year-in-art.markdown

```diff
@@ -0,0 +1,66 @@
+---
+layout: post
+title: "Animating the Year In Art"
+date: 2017-01-30 14:21
+author: eve
+categories: [javascript]
+---
+
+Artsy's end of year features are and annual chance to walk through highlights of the year while also exploring front-end experiments. Created in collaboration with UBS and designed by Owen Dodd, [The Year In Art 2016](http://artsy.net/year-in-art-2016) presents an interactive timeline of singular moments in art and culture over the past year.
+
+The piece opens with header animation, a series of transparent sliding boxes that presented a unique challenge. The finalized look is somewhat like a slinky-— a stack of containers that are stretched open from the bottom, and compress again as they reach the top of the viewport, collapsing inward without ever crossing outside the screen.
+
+Achieving this effect required animating elements in response both to the size of other elements in the viewport, and to the client’s scroll interactions.
```

Comment:
Great explanation of the requirements, it could use another sentence describing how this was challenging.